### PR TITLE
Desktop Portfolio :red_circle: 

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
- Adding CSS media queries
- Adding a desktop version to my portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,6 +92,7 @@ <h2>Profesional Art Printing Data</h2>
 
     <div id="secondCard">
         <h2>Profesional Art Printing Data</h2>
+        <h3 id="desktopHeading">Data Dashboard Healthcare</h3>
         <p>A daily selection of privately personalized reads; 
           no accounts or sign-ups required. has been the industry's standard</p>
           <div id="thirdTag">
```
Comment (OLIPLICHE) [inline index.html:98]: - [ ] Please make sure when the project is hovered on with the mouse the description and dark opacity disappears and a "See Project" button appears to the bottom of the image and takes the whole image width.
- [ ] Kindly make sure the "See Project" button takes you to the GitHub repository.

File: index.html [hunk 2]
```diff
@@ -103,6 +107,7 @@ <h2>Profesional Art Printing Data</h2>
 
     <div id="thirdCard">
         <h2>Profesional Art Printing Data</h2>
+        <h3 id="desktopHeading2">Website Protfolio</h3>
         <p>A daily selection of privately personalized reads;
```
Comment (OLIPLICHE) [inline index.html:111]: - [ ] Please the project description text should take 2 to 3 lines of text.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @afikanzoro , 

Good job so far!
There are some issues that you still need to work on to get your project approved, you are almost there!

## Highlights
- Linter result is green ✔️
- Commit messages are meaningful✔️
- The pull request have a descriptive title telling you what changes are inside ✔️
- The pull request have a short summary describing introduced changes in more detail ✔️
- node_modules folder is in the .gitignore ✔️

## Required Changes ♻️

- [ ] Please make sure that after the Section's title, there are only 4 to 5 lines of text describing the student's experience. 



- [ ] Kindly make sure there should be one input for Full name and this should take the space of both First Name and Last Name inputs (shown in the desktop version, Figma).
- This picture below is from your design

- The below is from Figma design




- [ ] Please the featured project description text should take 4 to 5 lines.
- [ ] `See Project` button should take you to the GitHub repository








 

- [ ] **CHECK THE COMMENTS UNDER THE REVIEW**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -1,10 +1,13 @@
+/* stylelint-disable selector-list-comma-newline-after */
+/* stylelint-disable indentation */
 /* stylelint-disable csstree/validator */
 /* stylelint-disable no-missing-end-of-source-newline */
 /* stylelint-disable no-descending-specificity */
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap');
 
+@media screen and (max-width: 992px) {
```
[the remark this answers is not in the document]
Comment (cyrilmuchemi) [reply styles.css:10]: @OLIPLICHE  On day 3 activity 4, we are told to ignore the figma design and implement only full name. Kindly confirm

[the remark this answers is not in the document]
Comment (cyrilmuchemi) [reply styles.css:10]: @OLIPLICHE  haven't I used min-width 992px in my desktop version, which is higher that 768? Kindly explain why you find this to be off-point

File: index.html (hunk 1 above)
Comment (cyrilmuchemi) [reply index.html:98]: @OLIPLICHE I don't understand the issue of disappearance as the hoover is working fine on my end. Can you kindly share a hint of the code you think I can use insead?

File: styles.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (OLIPLICHE) [reply styles.css:10]: Hi, you need to make sure that your desktop version looks like the one in the Figma design template that you chose([Project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements))

**Review by CollinsTatang — APPROVED** [review]
Hi @cyrilmuchemi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**